docs/source/_json/history_get.resp

```diff
@@ -0,0 +1,36 @@
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+[
+  {
+    "action": null, 
```

Comment:
@tisto I'm not sure if we can change this anymore. I can imaging that there are existing application that do something with the None/null value.

Maybe we should add this to plone.api (see https://github.com/plone/plone.api/issues/305) and use this method in here.